scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -193,11 +193,37 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
                 }
             };
 
+            function cleanEditorRanges(item) {
+                if (item.editor_state) {
+                    _.forEach(item.editor_state, (editorState) => {
+                        _.forEach(editorState.blocks, (block) => {
+                            if (block.data && block.data.inlineStyleRanges) {
+                                delete block.data.inlineStyleRanges;
+                            }
+                        });
+                    });
+                }
+            }
+
+            function restoreEditorRanges(item) {
+                if (item.editor_state) {
+                    _.forEach(item.editor_state, (editorState) => {
+                        _.forEach(editorState.blocks, (block) => {
+                            if (block.inlineStyleRanges) {
+                                block.data.inlineStyleRanges = block.inlineStyleRanges;
+                            }
+                        });
+                    });
+                }
+            }
+
             /**
              * Create a new version
              */
             $scope.save = function() {
```

Comment:
It would be much simpler and cleaner if you removed this code, and instead when you need the annotation when generating HTML, you could use `contentBlock.getInlineStyleAt()`